tools/mascot/src/main/java/dk/alexandra/fresco/tools/mascot/utils/FieldElementPrg.java

```diff
@@ -0,0 +1,53 @@
+package dk.alexandra.fresco.tools.mascot.utils;
```

Comment:
This package only contains the prg related stuff. Please rename to reflect this.

Reply:
Done.